sphinx/builders/html/__init__.py

```diff
@@ -1226,6 +1226,7 @@ def setup(app: Sphinx) -> Dict[str, Any]:
     app.add_config_value('html_search_scorer', '', None)
     app.add_config_value('html_scaled_image_link', True, 'html')
     app.add_config_value('html_baseurl', '', 'html')
+    app.add_config_value('html_codeblock_linenos', 'table', 'html', ENUM('table', 'inline'))
```

Comment:
Does anyone have a good name for this configuration?

Candidates:
* html_codeblock_linenos_style
* html_codeblock_line_numbers
* html_codeblock_line_numbers_style
* html_pygments_codeblock_linenos_style
* html_pygments_settings['codeblock_linenos']

Reply:
I certainly couldn't imagine what `html_codeblock_linenos` would do.

I prefer `html_codeblock_linenos_style` as it is shorter and easier to understand.

Or perhaps `html_pygments_linenos`.
This would be easier to convey the image of passing the pygments' linenos option directly to the `linenos` option.